### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,11 +1,107 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/134106079/17.2.3%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T590818)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 <!-- default file list -->
-*Files to look at*:
+# ASP.NET Web Forms Controls - How to implement a Theme Selector control shown in DevExpress Demos
```
Comment (arminal) [inline Readme.md]:
```suggestion
# ASP.NET Web Forms Controls - How to implement the Theme Selector control shown in DevExpress Demos
```

File: Readme.md [hunk 2]
```diff
@@ -1,11 +1,107 @@
...
+<!-- run online -->
+**[[Run Online]](https://codecentral.devexpress.com/t590818/)**
+<!-- run online end -->
 
+This example demonstrates how to implement a Theme Selector control as in [ASP.NET Web Forms Demos](https://demos.devexpress.com/ASP/).
```
Comment (arminal) [inline Readme.md]:
```suggestion
This example demonstrates how to implement the Theme Selector control as shown in [ASP.NET Web Forms Demos](https://demos.devexpress.com/ASP/). 
```

File: Readme.md [hunk 3]
```diff
@@ -1,11 +1,107 @@
...
+   <pages>
+     <controls>
+       <!-- ... -->
+       <add src="~/UserControl/ThemeParametersSelector.ascx" tagName="ThemeParametersSelector" tagPrefix="dx" />
+       <add src="~/UserControl/ThemeSelector.ascx" tagName="ThemeSelector" tagPrefix="dx" />
+     </controls>
+   </pages>
+   ```
+
+3. Once the user selects a theme, the code example writes the theme to cookies. To apply this theme, handle the `Application.PreRequestHandlerExecute` event in the *Global.asax* file as follows:
```
Comment (arminal) [inline Readme.md]:
```suggestion
3. Once a user selects a theme, the code example writes that theme to cookies [writes to cookies???]. To apply this theme, handle the `Application.PreRequestHandlerExecute` event in the *Global.asax* file as follows:
```

File: Readme.md [hunk 4]
```diff
@@ -20,102 +116,10 @@
...
 * [ThemeSelector.ascx.cs](./CS/UserControl/ThemeSelector.ascx.cs) (VB: [ThemeSelector.ascx.vb](./VB/UserControl/ThemeSelector.ascx.vb))
-<!-- default file list end -->
-# How to implement a Theme Selector control similar to DevExpress Demo
-<!-- run online -->
-**[[Run Online]](https://codecentral.devexpress.com/t590818/)**
-<!-- run online end -->
 
+## More Examples
+
+* [How to implement a Theme Selector control shown in DevExpress Demo (Old Style)](https://github.com/DevExpress-Examples/how-to-implement-a-theme-selector-control-similar-to-devexpress-demo-old-style-t504407)
```
Comment (arminal) [inline Readme.md]:
```suggestion
* [How to implement the Theme Selector control shown in the DevExpress Demo (Old Style)](https://github.com/DevExpress-Examples/how-to-implement-a-theme-selector-control-similar-to-devexpress-demo-old-style-t504407)
```

**Review by arminal — COMMENTED** [review]
#Articles_Grammar Please review usage differences between "a" and "the".

File: Readme.md (hunk 3 above)
Comment (SvetlanaMikheeva) [reply Readme.md]:
```suggestion
3. Once a user selects a theme, the code example writes that theme to cookies. To apply this theme, handle the `Application.PreRequestHandlerExecute` event in the *Global.asax* file as follows:
```

**Pull request merged**